strawberry/contrib/django/tests/test_model_type.py

```diff
@@ -0,0 +1,167 @@
+from typing import List
+
+import pytest
+
+import strawberry
+from strawberry.contrib.django.tests.models import TestModel
+from strawberry.contrib.django.type import model_type
+from strawberry.graphql import execute
+
+
+@pytest.mark.asyncio
+async def test_model_type():
+    @model_type(model=TestModel)
+    class TestModelType:
+        @strawberry.field
+        def extra_info(self, info) -> str:
+            return "more info"
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def model_type(root, info) -> TestModelType:
+            return TestModel(name="modeltype")
+
+    schema = strawberry.Schema(query=Query)
+
+    response = await execute(
+        query="""
+        query {
+            modelType {
+                name
+                extraInfo
+            }
+        }
+    """,
+        schema=schema,
+    )
+
+    assert response.data["modelType"]["name"] == "modeltype"
+    assert response.data["modelType"]["extraInfo"] == "more info"
+
+
+@pytest.mark.asyncio
+async def test_model_type_doesnt_override_type_fields():
```

Comment:
what about changing the name to:

```suggestion
async def test_model_type_can_override_type_fields():
```

?